docs/libcurl/symbols-in-versions

```diff
@@ -709,6 +709,8 @@ CURL_FORMADD_NULL               7.9.8
 CURL_FORMADD_OK                 7.9.8
 CURL_FORMADD_OPTION_TWICE       7.9.8
 CURL_FORMADD_UNKNOWN_OPTION     7.9.8
+CURL_GET_SSLVERSION             7.53.0
+CURL_GET_SSLVERSION_MAX         7.53.0
```

Comment:
I do not think these helper defines should be mentioned in symbols-in-versions.  They are not supposed to be used externally.